_NOT_EDITED_HERE.txt

```diff
@@ -0,0 +1,18 @@
+# Files and directories here are not edited in the docker.github.io repo.
```

Comment:
If this file is only used for automation, I suggest to either make it hidden (`.something`), or we can add a `hack/` directory to put everything in related to automation.